oauthproxy.go

```diff
@@ -1096,10 +1095,7 @@ func (p *OAuthProxy) CheckBasicAuth(req *http.Request) (*sessionsapi.SessionStat
 
 // isAjax checks if a request is an ajax request
 func isAjax(req *http.Request) bool {
-	acceptValues, ok := req.Header["accept"]
-	if !ok {
-		acceptValues = req.Header["Accept"]
-	}
+	acceptValues := req.Header.Values("Accept")
```

Comment:
Is this change functionally equivalent? ie are the headers case sensitive?

Reply:
The `Values` (since Go1.14) returns headers that are case insensitive.
https://golang.org/pkg/net/http/#Header.Values